tests/http_apis/run.sh

```diff
@@ -31,6 +31,7 @@ function run() {
     task_data=`cat $WORK_DIR/task.yaml.bak`
     rm $WORK_DIR/task.yaml.bak
     echo $task_data
+    sleep 2 # wait for embed HTTP service avaible
```

Comment:
can we add a simple backoff function to wait it ok?

Reply:
added backoff in https://github.com/pingcap/dm/pull/360/commits/aa3d4e77274bb56781582ba0fff369923e4edae2.